Makefile

```diff
@@ -573,9 +529,11 @@ $(eval $(call add_define,USE_COHERENT_MEM))
 $(eval $(call add_define,USE_TBBR_DEFS))
 $(eval $(call add_define,WARMBOOT_ENABLE_DCACHE_EARLY))
 
+ifeq ($(ARM_ARCH_MAJOR),7)
```

Comment:
The below `add_define` s can be moved to armv7-a-cpu.mk

Reply:
ok